src/backend/marsha/e2e/test_lti_bbb.py

```diff
@@ -360,7 +360,7 @@ def test_lti_bbb_create_enabled(page: Page, live_server: LiveServer, settings):
 
     expect(page.get_by_text("LTI link for this classroom:")).to_be_visible()
     invite_link_button = page.get_by_role(
-        "button", name="Invite someone with this link:"
+        "button", name="Invite a viewer with this link:"
```

Comment:
`a guest` ?

Reply:
The idea here, was to stick with BBB convention : users are viewers or moderators.
WDYT about `Invite a guest viewer with this link:` ?

Reply:
ok you can keep `a viewer` in that case.